Common/src/main/java/com/tabnineCommon/chat/TabnineChatProjectManagerListener.kt

```diff
@@ -36,6 +38,15 @@ class TabnineChatProjectManagerListener private constructor() : ProjectManagerLi
     fun start() {
         if (initialized.getAndSet(true)) return
 
+        if (StaticConfig.getTabnineEnterpriseHost().isPresent) {
```

Comment:
don't we have a `Channel.isOnprem` or something?
the channel check is stronger because its determined in compile time, so i'd consider checking it in addition to the available host check